tools/deploy_assisted_installer_configmap.py

```diff
@@ -112,6 +114,28 @@ def main():
         file=DST_FILE
     )
 
+def log_image_revision(image: str):
+    cmd = ["docker", "run", "--rm",  "quay.io/skopeo/stable", "inspect", f"docker://{image}"]
+    popen = subprocess.Popen(cmd,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT)
+    stdout, stderr = popen.communicate()
+
+    if popen.returncode != 0:
+        log.warn(f"error while getting image tag while running {cmd}, err: {stderr}")
+        return
+
+    try:
+        image_inspect = json.loads(stdout)
+    except ValueError as e:
+        log.warn(f"image inspect did not return an expected value while running {cmd}")
+        return
+
+    created = image_inspect.get("Created", None)
+    image_labels = image_inspect.get("Labels")
+    git_revision = image_labels.get("git_revision", None)
```

Comment:
if `image_labels` would be `None` you would get an exception, when you try to reach its attribute.

Reply:
so? 
all this means that in the case that label is not available it will print None and not break except logging this param is not going anywhere

Reply:
```
In [1]: image_inspect = dict()                                                  

In [2]: image_labels = image_inspect.get("Labels")                              

In [3]: git_revision = image_labels.get("git_revision", None)                   
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-2c6b86be4b8e> in <module>
----> 1 git_revision = image_labels.get("git_revision", None)

AttributeError: 'NoneType' object has no attribute 'get'
```

Reply:
I think in case the image has no Lable fieald we do want to fail, 
but ok.